inst/tests/tests.Rraw

```diff
@@ -12305,7 +12305,7 @@ DTout = data.table(
   value1 = 1:10,
   value2 = c("a", "b", "c", "d", "e", "f", "g", "h", "i", "j")
 )
-test(1866.6, melt(DT, measure.vars = patterns("^x", "^y", cols=names(DT))), DTout)
+test(1866.6, melt(DT, measure.vars = patterns("^x", "^y")), DTout)
```

Comment:
Isn't this a potentially breaking change vs. 1.15.0?

Reply:
yes, but I think it is appropriate
- there was only one test case
- specifying cols like this is redundant
- there is no other valid value for cols (other than names(DT))

Reply:
we need to be sure it is well mentioned in NEWS